core/src/main/java/ch/cyberduck/core/Scheme.java

```diff
@@ -110,6 +110,17 @@ public int getPort() {
             return 443;
         }
     },
+    smb {
+        @Override
+        public boolean isSecure() {
+            return false;
```

Comment:
Leads to the insecure connection alert? There is no secure alternative?

Reply:
Explicitly disabled in 953aa9b7. There is 
* SMB over QUIC (with TLS 1.3) that could be implemented.
* Windows Server 2022 and Windows 11 introduce AES-256-GCM and AES-256-CCM cryptographic suites for SMB 3.1.1 encryption [^1]. We could enable this in the dialect negotiation but it is not guaranteed the server does also support. As in this example with `SMB2_GLOBAL_CAP_ENCRYPTION`  from the integration tests

```
2023-08-24 22:48:31,727 [main] DEBUG ch.cyberduck.core.smb.SMBSession - Connected to ConnectionContext{
  serverGuid=34346430-6430-6438-3332-383300000000,
  serverName='localhost',
  negotiatedProtocol=NegotiatedProtocol{dialect=SMB_3_1_1, maxTransactSize=8388608, maxReadSize=8388608, maxWriteSize=8388608},
  clientGuid=379995c8-de62-4fcf-aa80-01af8a32de85,
  clientCapabilities=[SMB2_GLOBAL_CAP_DFS, SMB2_GLOBAL_CAP_LARGE_MTU, SMB2_GLOBAL_CAP_ENCRYPTION],
  serverCapabilities=[SMB2_GLOBAL_CAP_DFS, SMB2_GLOBAL_CAP_LEASING, SMB2_GLOBAL_CAP_LARGE_MTU],
  clientSecurityMode=1,
  serverSecurityMode=1,
  server='com.hierynomus.smbj.server.Server@32a2a6be'
}
```
 
[^1]: https://learn.microsoft.com/en-us/windows-server/storage/file-server/smb-security#smb-encryption